Library/Formula/gdb.rb

```diff
@@ -24,16 +24,17 @@ class Gdb < Formula
     sha256 "1c48758042eed1ba9357fdb98b49b5b819555d664f9bb876e35c8bbaad58b0ec" => :mavericks
   end
 
-  option "with-brewed-python", "Use the Homebrew version of Python"
+  deprecated_option "with-brewed-python" => "with-python"
+
+  option "with-python", "Use the Homebrew version of Python; by default system Python is used"
   option "with-version-suffix", "Add a version suffix to program"
   option "with-all-targets", "Build with support for all targets"
 
   depends_on "pkg-config" => :build
-  depends_on "readline"
-  depends_on "xz"
+  depends_on "python" => [:optional, "--universal"]
   depends_on "guile" => :optional
 
-  if build.with? "brewed-python"
+  if build.with? "python"
```

Comment:
Is this and the Requirement redundant if we're passing ` "--universal"` as part of the dependency above, or?